tests/framework/console/controllers/MigrateControllerTest.php

```diff
@@ -50,4 +50,34 @@ protected function getMigrationHistory()
         $query = new Query();
         return $query->from('migration')->all();
     }
+
+    public function testSessionMigration()
```

Comment:
This should be removed.
The test itself may exist, but it does not belong ot the `MigrateConrollerTest`